src/components/Select.vue

```diff
@@ -29,8 +43,20 @@ export default class Select extends Vue {
   @Prop(String)
   public name!: string;
 
+  @Prop(String)
+  public placeholder?: string;
+
   @Prop(Boolean)
   public required?: boolean;
+
+  @Prop(Array)
+  public options!: Option[];
+
+  public optionsVisible: boolean = false;
+
+  get selectedOption(): Option {
```

Comment:
Why is only the currently selected value stored and not `Option` object as a whole?
Wouldn't it be easier?

Reply:
This is similar to the html select element which also returns only the value of the currently selected option.